### DISCUSSION TITLE
Local Current Date

### DISCUSSION THREAD

**Discussion by impossibleapp** [supabase/supabase#28463] [Questions] [answered]
Right now it is 5.29 AM and today is 8th of Aug in India. When I query Now() or Current_Date, I am still being shown UK time and date (12 AM and 7th of Aug (GMT)).

Why is that? How to ensure that user's timezone is considered when fetching the current date and time.

![image](https://github.com/user-attachments/assets/8ff1e30c-f6ab-4480-81e1-9eabf5f2d7ce)

Please help!

**Comment by GaryAustin1** [comment, marked as answer]
https://supabase.com/docs/guides/database/postgres/configuration#managing-timezones
